### DISCUSSION TITLE
How do I mark an article as starred?

### DISCUSSION THREAD

**Discussion by slalaurette** [martinrotter/rssguard#977] [Q&A]
Starring articles is a feature I've been using extensively in other feed aggregators. I don't know how to use it in RSS Guard. 

I know it must exist, because one of the options when cleaning up the database is to delete starred articles. The problem is I don't know how to star an article. The context menu gives me the option of "switching importance" which doesn't look as the same thing. The keyboard shorcut list doesn't seem to include any options for starring an article. 

How am I supposed to mark an item as starred?

**Comment by martinrotter** [comment]
There is a columm for starred status of each article. You simply click into one of those two red areas, either directly in article list where after each click you see starred icon or in article preview panel.

![image](https://github.com/martinrotter/rssguard/assets/1255302/32d6d75f-a237-4cbf-820d-4f0ee6163a2a)

**Comment by slalaurette** [comment]
So, it's the same as "switch importance", right?
